cypress/integration/connect-wallet/test_connect_with_metamask.js

```diff
@@ -1,7 +1,10 @@
-describe('connect wallet: metamask', () => {
+describe.skip('connect wallet: metamask', () => {
   before(() => {
     cy.setupMetamask();
-    cy.changeMetamaskNetwork('localhost');
+  });
+
+  beforeEach(() => {
+    cy.acceptCookies();
```

Comment:
we can delete this spec file altogether since the use case is no longer needed in the test env.